### PR TITLE
Fix podspec warnings

### PR DESCRIPTION
**Pull request by soffes**
- Use new `resource_bundles` directive
- Remove deprecated `documentation` directive
- Upgrade FMDB to version 2.1 to fix its warnings

### CODE REVIEW

File: MapBox.podspec [hunk 1]
```diff
@@ -21,52 +21,7 @@ Pod::Spec.new do |m|
...
-      '--ignore', 'RMOpenSeaMapLayer.h',
-      '--ignore', 'RMOpenSeaMapSource.h',
-      '--ignore', 'RMPixel.h',
-      '--ignore', 'RMProjection.h',
-      '--ignore', 'RMTile.h',
-      '--ignore', 'RMTileImage.h',
-      '--ignore', 'RMTileSourcesContainer.h',
-    ]
-  }
+  m.resource_bundles = { 'MapBox' => ['MapView/Map/Resources/*'] }
```
Comment (bradfol) [inline MapBox.podspec:24]: one fix: rename bundle to `Mapbox` (lowercase b). The SDK throws an assertion when the it looks for the `Mapbox` bundle if not named correctly

**Comment by bradfol** [comment]
+1

**Comment by yannliang** [comment]
+1

File: MapBox.podspec (hunk 1 above)
Comment (soffes) [inline MapBox.podspec:24]: I tested this and what I have works just fine. Weird.

Comment (bradfol) [inline MapBox.podspec:24]: Try testing on latest `develop` branch. The commits that broke it happened on 12/17. Everything in the project was renamed from `MapBox` to `Mapbox`

Take a look at https://github.com/mapbox/mapbox-ios-sdk/commit/cffd593b5d4ff2503737a0157bf36b91b3391a11 specifically `RMMapView.m` line 429

https://github.com/mapbox/mapbox-ios-sdk/commit/cffd593b5d4ff2503737a0157bf36b91b3391a11#diff-cc99abd5af5db8f48b362bcac3e8c307L429

**Comment by incanus** [comment]
Thanks for this @soffes and sorry for the delay. I'm incorporating changes that encompass these myself now, aside from the FMDB upgrade as that needs more testing.

**Pull request closed without merging**